src/admin-views/aggregator/status.php

```diff
@@ -41,7 +41,7 @@
 			<td class="label"><?php esc_html_e( 'License Key', 'the-events-calendar' ); ?></td>
 			<td class="indicator <?php esc_attr_e( $indicator ); ?>"><span class="dashicons dashicons-<?php echo esc_attr( $indicator_icons[ $indicator ] ); ?>"></span></td>
 			<td><?php echo esc_html( $text ); ?></td>
-			<td><?php echo $notes; ?></td>
+			<td><?php echo $notes; // Escaping handled above. ?></td>
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$notes'

<sub>hash: 8f988e4b898610db14e916c7519e0311</sub>